tests/14_reduceOnlyBanks.spec.ts

```diff
@@ -0,0 +1,431 @@
+        await accountInit(program, {
+          marginfiGroup: marginfiGroup.publicKey,
+          marginfiAccount: user0AccountKeypair.publicKey,
+          authority: users[0].wallet.publicKey,
+          feePayer: users[0].wallet.publicKey,
+        })
+      ),
+      [user0AccountKeypair]
+    );
+
+    const user1AccountKeypair = Keypair.generate();
+    users[1].accounts.set(USER_ACCOUNT, user1AccountKeypair.publicKey);
+    await users[1].mrgnProgram.provider.sendAndConfirm(
+      new Transaction().add(
+        await accountInit(program, {
+          marginfiGroup: marginfiGroup.publicKey,
+          marginfiAccount: user1AccountKeypair.publicKey,
+          authority: users[1].wallet.publicKey,
+          feePayer: users[1].wallet.publicKey,
+        })
+      ),
+      [user1AccountKeypair]
+    );
+  });
+
+  it("(admin) Set bank to ReduceOnly, then restore to Operational - verifies state changes", async () => {
+    const bankKey = bankKeypairSol.publicKey;
+
+    
+    try {
+      // Set SOL bank to ReduceOnly
+      await groupAdmin.mrgnProgram.provider.sendAndConfirm!(
+        new Transaction().add(
+          await configureBank(groupAdmin.mrgnProgram, {
+            bank: bankKey,
+            bankConfigOpt: {
+              ...defaultBankConfigOptRaw(),
+              operationalState: {
+                reduceOnly: undefined,
+              },
+            },
+          })
+        )
+      );
+
+      let bank = await program.account.bank.fetch(bankKey);
+      assert.deepEqual(bank.config.operationalState, { reduceOnly: {} });
+
+      // Restore to Operational
+      await groupAdmin.mrgnProgram.provider.sendAndConfirm!(
+        new Transaction().add(
+          await configureBank(groupAdmin.mrgnProgram, {
+            bank: bankKey,
+            bankConfigOpt: {
+              ...defaultBankConfigOptRaw(),
+              operationalState: {
+                operational: undefined,
+              },
+            },
+          })
+        )
+      );
+
+
+      bank = await program.account.bank.fetch(bankKey);
+      assert.deepEqual(bank.config.operationalState, { operational: {} });
+    } finally {
+      // Ensure cleanup even if test fails
+      await groupAdmin.mrgnProgram.provider.sendAndConfirm!(
+        new Transaction().add(
+          await configureBank(groupAdmin.mrgnProgram, {
+            bank: bankKey,
+            bankConfigOpt: {
+              ...defaultBankConfigOptRaw(),
+              operationalState: {
+                operational: undefined,
+              },
+            },
+          })
+        )
+      );
+    }
+  });
+
+  it("(user 0) ReduceOnly collateral is worthless for new loans - should fail with RiskEngineInitRejected", async () => {
+    const user = users[0];
+    const userAccount = user.accounts.get(USER_ACCOUNT);
+
+
+    const depositAmountTokenA = 0.5;
+    const depositAmountTokenA_native = new BN(
+      depositAmountTokenA * 10 ** ecosystem.tokenADecimals
+    );
+
```

Comment:
send a health pulse (see 11_)  before/after the configBank and read that the: 
* `assetValue` was expected before (non-zero suffices here, or you can do the math: 0.5 * 10 * weight * (1-oracle_conf)), where weight is probably whatever is the default for the bank and oracle confidence is `ORACLE_CONF_INTERVAL * CONF_INTERVAL_MULTIPLE`) 
* declines to 0 to after
* the `assetValueMaint` is unchanged before/after